src/views/projects/Issue/New.svelte

```diff
@@ -51,9 +49,20 @@
           assignees: utils.stripDidPrefix(assignees),
           tags: tags,
         },
-        session.id,
+        sessionId,
       );
-      dispatch("create", result.id);
+
+      void router.push({
```

Comment:
I’d move this call after the try/catch block. If the call fails, it’s not an authentication error and the issue actually has been created so the error message would confuse the user.